docs/USING_FUNCTIONS.md

```diff
@@ -120,24 +122,21 @@ For example the [`verify` template](https://github.com/twilio-labs/function-temp
 The name of the Function gives away two pieces of information:
 
 1. If the name contains `.protected.` you'll want to make sure that the checkbox of _Check for valid Twilio signature_ is checked when you create the Function.
```

Comment:
This needs an update to say that it should be set to be a protected function. That checkbox is not a part of V2 functions.